### PR TITLE
#125 Fix clang compilation.

### PR DESCRIPTION
**Pull request by gautierbureau**

### CODE REVIEW

File: dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp [hunk 1]
```diff
@@ -341,665 +293,5 @@ TEST(SimulationTest, testSolverTRAPTestBeta) {
   ASSERT_DOUBLE_EQUALS_DYNAWO(yp[0], 1);
   ASSERT_DOUBLE_EQUALS_DYNAWO(z[0], 1);
 }
-/*
-TEST(SimulationTest, testSolverTRAPTestBetaUnstableRoot) {
```
Comment (rosiereflo) [inline dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp:345]: How come those tests were commented?

File: dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp [hunk 2]
```diff
@@ -141,54 +140,7 @@ boost::shared_ptr<Model> initModel(const double& tStart, Modeler modeler, bool s
   return model;
 }
 
-std::pair<boost::shared_ptr<Solver>, boost::shared_ptr<Model> > initSolverAndModel(std::string dydFileName, std::string iidmFileName,
```
Comment (rosiereflo) [inline dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp:144]: unused?

**Review by rosiereflo — CHANGES_REQUESTED** [review]

File: dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp (hunk 1 above)
Comment (gautierbureau) [reply dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp:345]: This test file was a copy paste of the one for SIM and only one of the multiple tests was added, and we did not see the rest was commented out. I am not even sure the testBeta is completly relevant as all those tests were designed for the SIM solver. We would need to have a closer look later

File: dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp (hunk 2 above)
Comment (gautierbureau) [reply dynawo/sources/Solvers/FixedTimeStep/SolverTRAP/test/TestBasics.cpp:144]: unsued because of the unsued tests

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**